pkg/endpoint/bpf.go

```diff
@@ -1347,6 +1350,11 @@ func (e *Endpoint) syncPolicyMapWithDump() error {
 		return err
 	}
 
+	// Skip full reconciliation if it has been performed recently.
+	if time.Since(e.policyMapLastFullReconciliation) < option.Config.PolicyMapFullReconciliationInterval {
```

Comment:
can we do this with a trigger and a min-interval instead?

Reply:
Not really. We want to run it on every `PolicyMapFullReconciliationInterval` and we want the visibility that `cilium status --all-controllers` gives us.